test_framework/musig.py

```diff
@@ -52,26 +52,15 @@ def sign_musig(priv_key, k_key, R_musig, P_musig, msg):
     """Construct a musig signature."""
```

Comment:
Update this comment to say "Construct a MuSig partial signature and return the s value."

Reply:
Done